rust/stackablectl/src/cmds/demo.rs

```diff
@@ -111,6 +112,13 @@ to specify operator versions."
 
     #[command(flatten)]
     namespaces: CommonNamespaceArgs,
+
+    #[arg(
+        long,
+        long_help = "Source the charts from either a OCI registry or from Nexus repositories.",
+        value_enum, default_value_t = Default::default()
+    )]
+    chart_source: ChartSourceTypeArg,
```

Comment:
suggestion: Add a doc comment here (short and long help), put attribute arguments on individual lines and drop the mention of Nexus.

```suggestion

    /// Source the charts from either a OCI registry or from index.yaml-based repositories.
    #[arg(
        long,
        value_enum,
        default_value_t = Default::default()
    )]
    chart_source: ChartSourceTypeArg,
```

Reply:
idea: Also, it might make sense to move this argument out of individual commands to `CommonRepoArgs` instead.

Reply:
The `CommonRepoArgs` are always displayed, even though they are not always relevant (e.g. when uninstalling operators). I only included `ChartSourceTypeArg` where it is actually used.

Reply:
See 32564c3